kbd.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: kbd
+  version: "2.7.1"
+  epoch: 0
+  description: Linux keyboard tools
+  copyright:
+    - license: GPL-2.0-or-later
+  resources:
+    cpu: 4
+    memory: 12Gi
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox-full
+      - coreutils
+      - linux-pam-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: cc4ad92a5e41689589e0a35e1c339b210679bb744dcec1b4088842e331756577
+      uri: https://www.kernel.org/pub/linux/utils/kbd/kbd-${{package.version}}.tar.gz
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --disable-nls \
+        --prefix=/usr \
+        --sysconfdir=/etc \
+        --datadir=/usr/share \
+        --htmldir=/usr/share/html/${{package.name}} \
+        --disable-static
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
```

Comment:
Do we need to strip?

Reply:
Yes and no. We don't add -g anywhere, and all binaries are built without debug symbols, so most of the time our strip pipeline is a no-op.

I sort of want to move away with having strip pipeline everywhere, and instead have melange force inject -g0 compiler flag (or be tunable, such that if you want a debug rebuild you can do that without changing .yaml; and for software to opt-in into debug symbols if they need them - like openssl packaging does it).

Reply:
Done